test/unit/service/JobRoleUITest.js

```diff
@@ -0,0 +1,31 @@
+const { Builder, By, Key, until } = require('selenium-webdriver');
+
+
+async function JobRolesUITest() {
+
+  const driver = await new Builder().forBrowser('chrome').build();
+
+  try {
+
+    await driver.get('http://localhost:3000/job_roles');
```

Comment:
pass the base url, i.e. http://localhost:3000, in as an environment variable so we can run these against AWS URL in future